istio-control/istio-config/templates/deployment.yaml

```diff
@@ -181,9 +181,28 @@ spec:
             readOnly: true
           - name: envoy-config
             mountPath: /var/lib/istio/galley/envoy
+          {{- if .Values.global.sds.enabled }}
+          - name: sds-uds-path
+            mountPath: /var/run/sds
+            readOnly: true
+          - name: istio-token
+            mountPath: /var/run/secrets/tokens
+          {{- end }}
 {{- end }}
 
       volumes:
+      {{- if .Values.global.sds.enabled }}
+      - hostPath:
+          path: /var/run/sds
+        name: sds-uds-path
+      - name: istio-token
+        projected:
+          sources:
+          - serviceAccountToken:
+              audience: {{ .Values.global.sds.token.aud }}
+              expirationSeconds: 43200
```

Comment:
why this expirationSeconds magic number?

Reply:
From the old installer and this is what we've been using. I'd rather not changing any constants in this PR.

Reply:
This is 12 hours. We made this the default JWT lifetime. I think it's a balance of making the JWT short-lived VS. not causing too much JWT rotation overhead.